src/test/java/com/google/sps/SchedulingTest.java

```diff
@@ -58,12 +73,17 @@ public void testDoPost() throws Exception {
         verify(request, atLeast(1)).getParameter("tutorID");
         verify(request, atLeast(1)).getParameter("start");
         verify(request, atLeast(1)).getParameter("end");
+        verify(request, atLeast(1)).getParameter("year");
```

Comment:
should these be checking exactly 1 instead of at least?

Reply:
That would make it more accurate, yes. I added those changes in a new commit. Thanks!